index.html

```diff
@@ -21,156 +21,161 @@
 
   <body>
     <div class="header">
+      <a href="#">
+        <img
+          src="images/bikes-for-refugees_logo.jpg"
+          alt="Logo for Bikes for Refugees Scotland"
+          class="header__logo"
+        />
+      </a>
+
+      <div>
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Home</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">About Us</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">News & Events</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </div>
 
-        <a href="#">
-          <img
-            src="images/bikes-for-refugees_logo.jpg"
-            alt="Logo for Bikes for Refugees Scotland"
-            class="header__logo"
-          />
-        </a>
-
-        <div>
-          <ul class="navigation__list">
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Home</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">About Us</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">News & Events</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Contact Us</a>
-            </li>
-          </ul>
-        </div>
-
-        <button>Donate Now</button>
-
+      <button>Donate Now</button>
```

Comment:
on hover of the button, it becomes blue, can you make it orange as required, thanks
![Screenshot 2023-09-26 at 09 19 22](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/07eb0d9a-d804-4ed1-bd73-2afc7ff061b8)

Reply:
Thank you for your help. I have reviewed and made corrections to the areas you pointed out. If you find something  Idid not do correctly just let me know. Thank you for your time

Reply:
![Screenshot 2023-09-27 at 10 12 08](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/c5b71730-3058-4ad7-98b1-130c6dd1fb56)

When I hover on this button, the background should be white and the text should be orange, see a picture of what it should look like
![Screenshot 2023-09-27 at 10 13 37](https://github.com/CodeYourFuture/bikes-for-refugees/assets/13808341/f5f0cb1d-9462-4a48-928d-04a41b605c4f)